.github/dependabot.yml

```diff
@@ -1,8 +1,8 @@
 # Generated by: tyk-ci/wf-gen
-# Generated on: Fri  3 Sep 12:23:14 UTC 2021
+# Generated on: Fri Jan 14 17:45:41 UTC 2022
 
 # Generation commands:
-# ./pr.zsh -title [TT-2932] systemd service restore for deb -base releng/upgrades -branch releng/upgrades -repos tyk-sink,tyk-identity-broker,tyk-pump,raava -p
```

Comment:
@ermirizio what is the reason to delete tyk-sink, tyk-pump and raava from repos?

Reply:
well that's just the last command executed, being override by the new command, doesn't mean we deleted anything but instead which repos are we doing sync over. 
In this case I choose only to sync against the current repo TIB and it's fine, there's no policy about when to sync, it's a responsibility of each squad to sync against the latest Releng code before releasing.